daemon/src/projection.rs

```diff
@@ -261,11 +265,17 @@ impl Cfd {
                 (CfdState::ContractSetup, vec![])
             }
             ContractSetupCompleted { dlc } => {
-                self.details.tx_url_list.push(TxUrl::new(
-                    dlc.lock.0.txid(),
+                // Lock transaction has shared output, but it likely has change
+                // output too that can be highlighted
```

Comment:
It's more that we can highlight the shared output, but cannot discern between our change outputs and the other party's.

Reply:
changed the comment.